### DISCUSSION TITLE
in mac iterm2, how to enable mouse double-click to select text to copy?

### DISCUSSION THREAD

**Discussion by haiwu** [zellij-org/zellij#3402] [General]
In regular mac iterm2, I could use mouse to double-click to select text, and that would be copied auto. Is it possible to do the same inside Zellij, which is inside mac iterm2? 

It does not work by default.

**Comment by imsnif** [comment]
You can if you shift-click, but then it will also mark the pane borders (since shift-clicking means you're bypassing Zellij's mouse-handling temporarily). I want to implement this ability natively, but have not yet gotten around to it. No ETA I'm afraid.

**Comment by haiwu** [comment]
Thank you! Turns out in my case it is option-click (I might have some custom remapping then). Yes it does mark the pane border if clicking 3 times. This is a very frequent operation I have to do, so it would be a very useful feature for me to look forward to later.
